easybuild/easyconfigs/l/libtree/libtree-2.0.0-GCCcore-10.3.0.eb

```
@@ -0,0 +1,36 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'CMakeMake'
+
+name = 'libtree'
+version = '2.0.0'
+
+homepage = 'https://github.com/haampie/libtree'
+description = """libtree is a tool that turns ldd into a tree, explains why
+ shared libraries are found and why not and optionally deploys executables and
+ dependencies into a single directory"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+github_account = 'haampie'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['099e85d8ba3c3d849ce05b8ba2791dd25cd042a813be947fb321b0676ef71883']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake', '3.20.1'),
+    ('cxxopts', '3.0.0'),
+    ('ELFIO', '3.9'),
+    ('termcolor', '2.0.0'),
+]
+
+sanity_check_paths = {
+    'files': ["bin/%(name)s"],
+    'dirs': ['bin', 'share']
```

```suggestion
    'dirs': ['share']
```
(no need to check dir `bin` if already checking file inside it)